app/helpers/books_helper.rb

```diff
@@ -0,0 +1,12 @@
+module BooksHelper
+  def book_rate book = Book.new
+    return 0 unless rate = book.reviews.average(:rate)
```

Comment:
```suggestion
    return {} unless rate = book.reviews.average(:rate)
```
để thế này đi, cho thống nhất với hash bên dưới :v hoặc để {rate: 0, want_rate: 5}

Reply:
Ok nhé anh !